src/fides/api/models/privacy_preference.py

```diff
@@ -328,95 +227,28 @@ class ServedNoticeHistory(ConsentReportingMixin, Base):
-        cls: Type[ServedNoticeHistory],
-        db: Session,
-        *,
-        data: dict[str, Any],
-        check_name: bool = False,
-    ) -> Tuple[ServedNoticeHistory, LastServedNotice]:
-        """Create a ServedNoticeHistory record and then upsert the LastServedNotice record.
-
-        If separate LastServedNotice records exist for both a verified provided identity and a fides user device
-        id provided identity, consolidate when the notice was last served into a single record.
-
-        There is only one LastServedNotice for each PrivacyNotice/ProvidedIdentity.
-        """
-        (
-            privacy_notice_history,
-            tcf_field,
-            tcf_val,
-        ) = _validate_before_saving_consent_history(db, data)
-
-        if tcf_field:
-            data["tcf_version"] = CURRENT_TCF_VERSION
-
-        created_served_notice_history = super().create(
-            db=db, data=data, check_name=check_name
-        )
-
-        last_served_data = {
-            "provided_identity_id": created_served_notice_history.provided_identity_id,
-            "served_notice_history_id": created_served_notice_history.id,
-            "fides_user_device_provided_identity_id": created_served_notice_history.fides_user_device_provided_identity_id,
-        }
-
-        # Add additional data to store the "historical" version of the consent component that was served.
-        # Either the privacy notice history record or the TCF version
-        if privacy_notice_history:
-            last_served_data["privacy_notice_history_id"] = privacy_notice_history.id
-            last_served_data[
-                "privacy_notice_id"
-            ] = privacy_notice_history.privacy_notice_id
-
-        elif tcf_field:
-            last_served_data[tcf_field] = tcf_val
-            last_served_data["tcf_version"] = CURRENT_TCF_VERSION
-
-        upserted_last_served_notice_record = upsert_last_saved_record(
-            db,
-            created_historical_record=created_served_notice_history,
-            current_record_class=LastServedNotice,
-            current_record_data=last_served_data,
-        )
-        assert isinstance(
-            upserted_last_served_notice_record, LastServedNotice
-        )  # For mypy
-
-        return created_served_notice_history, upserted_last_served_notice_record
 
+class DeprecatedPrivacyPreferenceHistory(ConsentReportingMixin, Base):
+    """
+    ***DEPRECATED*** in favor of PrivacyPreferenceHistoryV2
 
-class PrivacyPreferenceHistory(ConsentReportingMixin, Base):
-    """The DB ORM model for storing PrivacyPreferenceHistory, used for saving
+    The DB ORM model for storing PrivacyPreferenceHistory, used for saving
     every time consent preferences are saved for reporting purposes.
+
+    Soon to be deprecated in favor of PrivacyPreferenceHistoryv2
     """
 
+    @declared_attr
+    def __tablename__(self) -> str:
+        return "privacypreferencehistory"
```

Comment:
Prefixed the old models with Deprecated* until I remove them entirely but tablenames have stayed the same.